hack/templates/dev-builds-wo-deployer.yaml

```diff
@@ -0,0 +1,214 @@
+apiVersion: v1
+kind: "Template"
+metadata:
+  name: logging-dev-build-wo-deployer-template
+  annotations:
+    description: "Template for creating local builds of logging components from source."
+    tags: "infrastructure"
+labels:
+  logging-infra: development
+  provider: openshift
+  component: development
+objects:
+- apiVersion: v1
+  kind: ImageStream
+  metadata:
+    labels:
+      build: logging-elasticsearch
+    name: logging-elasticsearch
+  spec: {}
+- apiVersion: v1
+  kind: ImageStream
+  metadata:
+    labels:
+      build: logging-fluentd
+    name: logging-fluentd
+  spec: {}
+- apiVersion: v1
+  kind: ImageStream
+  metadata:
+    labels:
+      build: logging-kibana
+    name: logging-kibana
+  spec: {}
+- apiVersion: v1
+  kind: ImageStream
+  metadata:
+    labels:
+      build: logging-curator
+    name: logging-curator
+  spec: {}
+- apiVersion: v1
+  kind: ImageStream
+  metadata:
+    labels:
+      build: logging-auth-proxy
+    name: logging-auth-proxy
+  spec: {}
+- apiVersion: v1
+  kind: ImageStream
+  metadata:
```

Comment:
I would not think so, but I'm having issues trying to get a passing build and not sure I have removed all that I wold normally remove from files